chunks/lang-go/chunk.yaml

```diff
@@ -2,3 +2,6 @@ variants:
   - name: "1.17.5"
```

Comment:
1.17.5 is not referenced anywhere, so we should remove it.

Reply:
not sure - if we want to offer a "features" like feature, this would still be handy. We could also remove it now and add it back in later.

Reply:
Removing it now and adding it back later is the best way in my opinion. Leaving it in "just in case" will increase build times without providing value in return and we might not actually need it in the future.

Reply:
done